include/Mixer.h

```diff
@@ -151,7 +151,7 @@ class LMMS_EXPORT FxMixer : public Model, public JournallingObject
 
 	QString nodeName() const override
 	{
-		return "fxmixer";
+		return "mixer";
 	}
 
 	FxChannel * effectChannel( int _ch )
```

Comment:
I think `FxChannel`, `FxRoute`, `FxLine` should all use their `Fx`. There should be no `Fx` or `fx` in this file anymore.